scripts/index-services.py

```diff
@@ -94,7 +98,7 @@ def do_index(search_api_url, search_api_access_token, data_api_url, data_api_acc
         pool = multiprocessing.Pool(10)
         mapper = pool.imap
 
-    indexer = ServiceIndexer(search_api_url, search_api_access_token, index)
+    indexer = ServiceIndexer(search_api_url, search_api_access_token, index, frameworks)
```

Comment:
I don't think it does: https://github.com/alphagov/digitalmarketplace-api/blob/master/app/main/views/services.py#L47